src/Symfony/Bundle/WebProfilerBundle/Tests/Resources/MinifyTest.php

```diff
@@ -23,8 +23,12 @@ class MinifyTest extends TestCase
     public function testNoSingleLineComments()
     {
         $dir = \dirname(__DIR__, 2).'/Resources/views/Profiler';
-        $message = 'There cannot be any single line comment in this file. Consider using multiple line comment. ';
-        $this->assertTrue(2 === substr_count(file_get_contents($dir.'/base_js.html.twig'), '//'), $message);
-        $this->assertTrue(0 === substr_count(file_get_contents($dir.'/toolbar.css.twig'), '//'), $message);
```

Comment:
what about this CSS files ?

Reply:
I don't think this applies, `//` is not a valid CSS comment and CSS only supports the `/* ... */` comment syntax.